src/get-started/java-binding.md

```diff
@@ -303,15 +351,18 @@ The `Transaction#execute` method describes the operations that are applied to th
 current storage state when the transaction is executed. Exonum passes
 an [execution context][transaction-execution-context] as an argument,
 which provides a `Fork` – a view that allows performing modifying
-operations; and some information about the corresponding transaction message:
-its SHA-256 hash that uniquely identifies it, and the author’s public key.
+operations; the service instance name and numeric ID; and some information
+about the corresponding transaction message: its SHA-256 hash that uniquely 
+identifies it, and the author’s public key.
 A service schema object can be used to access data collections of this service.
 
 Also, `Transaction#execute` method may throw `TransactionExecutionException`
 which contains a transaction error report. This feature allows users to notify
 Exonum about an error in a transaction execution whenever one occurs.
 It may check the preconditions before executing a transaction and either
 accepts it or throws an exception that is further transformed into an Exonum
+<!-- fixme: Shall we reference the new core enum (ExecutionStatus + ExecutionError)?
```

Comment:
Protobuf message *could* make sense, but I'm not sure what's a good way to link to it (other than to the source code).

Reply:
We could actually link it, because our 'common' module builds the message classes.

But the bigger problem is that most protobuf _messages_ have bad documentation (and hence the generated classes), see https://jira.bf.local/browse/ECR-3731

Reply:
That's how it looks like:
![зображення](https://user-images.githubusercontent.com/7531824/70239129-62885080-1773-11ea-9651-2534ffac1b78.png)